avocado/utils/linux_modules.py

```diff
@@ -96,7 +101,22 @@ def loaded_module_info(module_name):
     :rtype: dict
     """
     l_raw = process.system_output('/sbin/lsmod')
-    return parse_lsmod_for_module(l_raw, module_name)
+    modinfo_dic = parse_lsmod_for_module(l_raw, module_name)
+    output = process.system_output("/sbin/modinfo %s" % module_name)
+    if output:
+        param_list = []
+        for line in output.splitlines():
```

Comment:
I'm not sure what you mean by `pci_address`, but I'm fine with keeping it the way it is. It was just a question whether you plan to do some updates on this in the future, which would be only possible with params as dict...